bulk/jobs.go

```diff
@@ -0,0 +1,126 @@
+package bulk
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"strconv"
+
+	sfdc "github.com/g8rswimmer/go-sfdc"
+	"github.com/g8rswimmer/go-sfdc/session"
+)
+
+// Parameters to query the bulk jobs.
+//
+// IsPkChunkingEnabled will filter jobs with PK chunking enabled.
+//
+// JobType will filter jobs based on job type.
+type Parameters struct {
+	IsPkChunkingEnabled bool
+	JobType             JobType
+}
+
+type jobResponse struct {
+	Done           bool       `json:"done"`
+	Records        []Response `json:"records"`
+	NextRecordsURL string     `json:"nextRecordsUrl"`
+}
+
+// Jobs presents the response from the all jobs request.
+type Jobs struct {
+	session  session.ServiceFormatter
+	response jobResponse
+}
+
+func newJobs(session session.ServiceFormatter, parameters Parameters) (*Jobs, error) {
```

Comment:
where is this used?

Reply:
This is used for the `all jobs` API and in the `bulk.go` file 
https://developer.salesforce.com/docs/atlas.en-us.api_bulk_v2.meta/api_bulk_v2/get_all_jobs.htm

Reply:
Oh I see it now.  Sorry - wasn't able to search the usage of the function call before for some reason.